### ISSUE TITLE
Touch input not working

### ISSUE THREAD

**Issue by raminf** [migeran/libgodot_project#4]
I'm evaluating whether I can use `libgodot` as a canvas for mini-games inside a SwiftUI app on iOS/iPadOS.

The .pck files are loading fine and showing, but no touch input is being processed. Godot-defined interactive elements don't appear to work.

I've tried testing with two separate Godot 4 projects:

- https://github.com/Revelnaut/blokki
- https://github.com/Goldenlion5648/Godot4SignalsDemo

Not sure whether if there's something else I should be setting in the exporter.

Another thing that isn't clear is how to best communicate between the embedded Godot view and the native app (and vice-versa). For example, to pass a username down into the Godot view, and have the scores from the mini-games bubble back up to the main application. I can file that as a separate issue if easier to track.

**Comment by kisg** [comment]
Thank you for your interest. We are preparing a larger update where all inputs (touch, keyboard, mouse) will be handled properly.

We will update this issue once the update is ready. 

I moved your other question to a separate issue.

**Comment by raminf** [comment]
OK, thanks. Looking forward to trying it out. 

I like the direction it's going vs. the Unity embedding model that only works full-screen. FWIW, in my test app, I'm embedding it in SwiftUI inside the Tab item of a tab bar.
